modules/analyzing-all-violations.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operating/respond-to-violations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="analyzing-all-violations_{context}"]
+= Analyzing all violations
+
+By viewing the *Violations* page, you can analyze all violations and take corrective action.
+
+.Procedure
+
+. In the {product-title-short} portal, click *Violations*.
+. Choose the appropriate method to view the violations from the drop-down list, which is in the upper left of the page:
+** To display the findings for application workloads, select *Applications view*
+** To display the findings for platform components in OpenShift, select *Platform view*.
+** To display the findings for application workloads and platform components simultaneously, select *Full view*.
+. Optional: Choose the appropriate method to re-organize the information in the *Violations* page:
+** To sort the violations in ascending or descending order, select a column heading.
+** To filter violations, use the filter bar.
+** To see more details about the violation, select a violation in the *Violations* page.
+. Optional: To exclude a policy from the violations list, perform the following steps:
+.. Locate the policy that you want to exclude.
+.. Click the overflow menu, {kebab}, and then select *Exclude deployment from policy*.
```

Comment:
I think per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#inserting-reusable-images-inline), you don't need to include the commas:
```suggestion
.. Click the overflow menu {kebab} and then select *Exclude deployment from policy*.
```